### ISSUE TITLE
GitHub packages

### ISSUE THREAD

**Issue by senyast4745** [Niflheim/Stockfish-Java#1]
Hello! Could you add jar to the `GitHub/Maven` artifactory so that your project can be conveniently used in other projects? It would be cool!

**Comment by Niflheim** [comment]
I have some breaking changes in private git that haven't been pushed to this repository, but I will definitely look into getting it added.

**Comment by senyast4745** [comment]
Thank you very much for your reply.  I have one more small question.  You have **API** support for *Query* **moves**, but as far as I understand from the code, the **moves** are still not used.

**Comment by Niflheim** [comment]
It's been a while since I've last touched this code, are you perhaps referencing the this void method in StockfishClient.java?
```
public class StockfishClient {
    ...
    public void submit(Query query) {
        submit(query, null);
    }
    ...
}
```
This was originally meant for controlling the Stockfish instance after it was instantiated (such as tweaking options per instance thus not needing a callback) but I never really got around to updating the Query.java class to actually make it relevant. 

If what you're referencing is something different, please link it so I can take a look!

**Comment by senyast4745** [comment]
I'm talking about the `Query.Builder.setMove (String move);` method.  It seems to me that passing **FEN** is not always convenient, and I thought that you were going to make a method that would take just the player’s move as a string like a2a3. It looks easier)

**Comment by Niflheim** [comment]
If I remember correctly you do pass in String moves like e2e4 into that method, but I will take a look at this more when I have time.

**Comment by senyast4745** [comment]
Yes, you’re right, I pass a string there like a2a3, but judging by the code, this method is not used anywhere.

```
public Query(QueryType type, String fen, int difficulty, int depth, long movetime) {
        this.type = type;
        this.fen = fen;
        this.difficulty = difficulty;
        this.depth = depth;
        this.movetime = movetime;
    }
```
